via/via_system_linux.cpp

```diff
@@ -89,6 +90,8 @@ int ViaSystemLinux::RunTestInDirectory(std::string path, std::string test, std::
             err = chdir(orig_dir);
         }
     }
+    if (err_code && err_code != 1)
```

Comment:
Nit: Wrap conditional code in braces.

Reply:
Commit 71c075d9e59071ceffffe55b6a56bf18c24a58ad addressed this.